js/emoji_util.js

```diff
@@ -16,9 +16,83 @@
         this.img_sets.apple.path = '/images/emoji/apple/';
         this.replace_mode = 'img';
     };
+
+    EmojiConvertor.prototype.getCountOfAllMatches = function(str, regex) {
+        var match = regex.exec(str);
+        var count = 0;
+
+        while (match) {
```

Comment:
Caveat emptor: this will loop forever if the `/g` is ever dropped from the regex. Probably not gonna happen, but ya never know...